docs/_posts/2016-10-10-client-startup-and-configuration.adoc

```diff
@@ -0,0 +1,145 @@
+
+=== [auth]
+
+This section is required for connection to an SOTA server that implements authentication. It needs to be the first section of the config file. If you are not using authentication, or if you are using RVI transport for authentication, you should *delete* this section entirely.
+
+----
+[auth]
+server = "https://auth-plus.gw.prod01.advancedtelematic.com" <1>
+client_id = "bf66425f-d4d6-422b-b510-7c7f178af9fe" <2>
+client_secret = "hr8nEWzQc9" <2>
+credentials_file = "/opt/sota/credentials.toml" <3>
+----
+<1> The URL of the auth server.
+<2> A unique client ID and secret for this device.
+<3> If this value is defined, SOTA client will check this file for auth credentials first, and use the credentials it finds there. If the file does not exist, it will be created with the `client_id` and `client_secret` above.
+
+=== [core]
+
+This is simply the URL of the core SOTA update server.
+
+----
+[core]
+server = "https://sota-core.gw.prod01.advancedtelematic.com"
+----
+
+=== [device]
+
+This section contains device-specific configuration.
+
+----
+[device]
+uuid = "123e4567-e89b-12d3-a456-426655440000" <1>
+vin = "" <2>
+system_info = "system_info.sh" <3>
+polling_interval = 10 <4>
+packages_dir = "/tmp/" <5>
+package_manager = "deb" <6>
+certificates_path = "/tmp/sota_certificates" <7>
+----
+<1> The UUID of the device. This is assigned by ATS Garage upon device creation, and should not be changed.
+<2> The device's VIN, if it has one. Deprecated feature; this value is not used by the current version of ATS Garage.
+<3> The script to use to gather system information.
+<4> The frequency, in seconds, with which the SOTA client should poll the server for updates.
+<5> The location SOTA Client should use for temporary package storage until they are processed by the software loading manager.
+<6> The software loading manager backend to use. Possible values are `deb`, `rpm`, and `off`. If an external software loading manager is in use, this should be set to `off`.
+<7> The certificate authorities SOTA Client trusts. Defaults are taken from Mozilla Servo.
+
+=== [network]
+
+This section contains network configuration information. Some values may be ignored if the corresponding communication methods aren't enabled.
+
+----
+[network]
+http_server = "127.0.0.1:8080" <1>
+socket_commands_path = "/tmp/sota-commands.socket" <2>
+socket_events_path = "/tmp/sota-events.socket" <3>
+websocket_server = "https://sota-core.gw.prod01.advancedtelematic.com" <4>
+rvi_edge_server = "127.0.0.1:9080" <5>
+----
+<1> The path to the http-only core server, if the http gateway is enabled in the [gateway] section.
```

Comment:
Oops, that's an oversight on my part. I'll get it fixed. As for updating in the client repo, the idea was to consolidate the docs in one place—so I was going to make a PR removing the redundant info from the client repo once this one is approved.